src/Microsoft.ML.CpuMath/Sse.cs

```diff
@@ -401,10 +401,10 @@ public static void SdcaL1UpdateDense(float primalUpdate, int count, ReadOnlySpan
             }
         }
 
-        public static void SdcaL1UpdateSparse(float primalUpdate, int count, ReadOnlySpan<float> src, ReadOnlySpan<int> indices, float threshold, Span<float> v, Span<float> w)
+        public static void SdcaL1UpdateSparse(float primalUpdate, int count, ReadOnlySpan<float> source, ReadOnlySpan<int> indices, float threshold, Span<float> v, Span<float> w)
```

Comment:
I don't know what `v` and `w` stand for throughout, but in some future PR someone might want to make them more descriptive...